Demo/Sources/Recycling/ListViews/Settings/SettingsViewDemoView.swift

```diff
@@ -53,11 +53,10 @@ class SettingsViewDemoView: UIView, Demoable {
     ]
 
     private lazy var settingsView: SettingsView = {
-        let settingsView = SettingsView(viewTitle: nil)
+        let settingsView = SettingsView(viewTitle: nil, versionText: "FinniversKit Demo", logoImage: .brandLogoSimple)
```

Comment:
`textVersion`

Reply:
or just `text` 🤷‍♂️

Reply:
The purpose of this label is to show current app version to the users, so it was called versionText initially

Reply:
Here we just show that this is a Demo app for users of FinniverseKit Demo
p.s. imho this naming is good enough

Reply:
aah, i see. for some reason, i thought it was meant to imply a `version of the text` but here it actually means version, as in the apps version 
👍